### PR TITLE
Sample and demo data

### LINKED ISSUE #885: Document using sample data buckets in Cloud
InfluxDB Cloud users will soon have the option to enable sample data buckets. We need to document how to access them in the UI and use those buckets in  queries.

Another thought – we have various options for sample data (sample data buckets, sample data repos, etc.). We probably need a sample data page that lists each of the sample options and how to use them.

### PR DESCRIPTION
**Pull request by sanderson**
Closes #885 

This introduces content to accompany demo data buckets in InfluxDB Cloud. It also consolidates sample data sets into a single page.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by sanderson** [comment]
Add a link on the getting started doc that goes to the sample data page.

File: content/v2.0/write-data/sample-data/_index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Sample data
```
Comment (ebb-tide) [inline content/v2.0/write-data/sample-data/_index.md:2]: Can we change this and all subsequent references to Demo Data please?

File: content/v2.0/write-data/sample-data/_index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+Use sample data to familiarize yourself with time series data and InfluxDB.
+InfluxData provides many sample time series datasets to use with **InfluxDB** and **InfluxDB Cloud**.
+
+{{< children >}}
+
+---
+
+### Air sensor sample data
```
Comment (ebb-tide) [inline content/v2.0/write-data/sample-data/_index.md:20]: I don't think we're using these demo data sets anymore :( just website monitoring using telegraf to ping a number of urls

File: content/v2.0/write-data/sample-data/_index.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/write-data/sample-data/_index.md:2]: Good suggestion, but there was a few reasons for doing this:

1. Findability - people search for "sample data" more than "demo data."
2. Differentiation – This PR includes pages for both "Sample data" and "Demo data":
    - **Sample data:** Semi-static data sets we provide in the documentation to highlight/walkthrough specific features.
    - **Demo data:** Buckets provided in InfluxDB for introducing and demo-ing the product.

File: content/v2.0/write-data/sample-data/_index.md (hunk 2 above)
Comment (sanderson) [reply content/v2.0/write-data/sample-data/_index.md:20]: We still provide and use sample data sets in the documentation and a few other places.

Comment (ebb-tide) [reply content/v2.0/write-data/sample-data/_index.md:20]: ah ok ! :)

File: content/v2.0/write-data/sample-data/_index.md (hunk 1 above)
Comment (ebb-tide) [reply content/v2.0/write-data/sample-data/_index.md:2]: ok! sorry I thought this was documentation of demodata feature

Comment (sanderson) [reply content/v2.0/write-data/sample-data/_index.md:2]: No worries. This PR both consolidates the existing sample data sets into a single doc and introduces a new demo data doc.

**Comment by russorat** [comment]
@sanderson & @timhallinflux I don't think we should document sample datasets that are not in the format of an InfluxDB Template. We don't have to store them in the community-templates repo (although i think we should tbh), but they should follow similar patterns as the other ones (readme instructions, use influx pkg cmd, etc.)

File: content/v2.0/write-data/sample-data/demo-data.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+## Explore demo data
+Use the [Data Explorer](https://v2.docs.influxdata.com/v2.0/visualize-data/explore-metrics/)
+to query and visualize data in demo data buckets.
+
+In the navigation menu on the left, click **Explore (Data Explorer)**.
+
+{{< nav-icon "explore" >}}
+
+## Visualize demo data
+When adding a demo data bucket, InfluxDB Cloud provides a pre-built dashboard for the demo data.
```
Comment (kelseiv) [inline content/v2.0/write-data/sample-data/demo-data.md:47]: View demo data in a pre-built dashboard.

File: content/v2.0/write-data/sample-data/demo-data.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## Explore demo data
+Use the [Data Explorer](https://v2.docs.influxdata.com/v2.0/visualize-data/explore-metrics/)
+to query and visualize data in demo data buckets.
+
+In the navigation menu on the left, click **Explore (Data Explorer)**.
+
+{{< nav-icon "explore" >}}
+
+## Visualize demo data
```
Comment (kelseiv) [inline content/v2.0/write-data/sample-data/demo-data.md:46]: bc folks can also "visualize" in Data Explorer, how about combining "Explore and visualize demo data", and then say do one of the following:
- Open Data Explorer
- Open Dashboard

File: content/v2.0/write-data/sample-data/demo-data.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+1. In the navigation menu on the left, click **Boards (Dashboards)**.
+
+    {{< nav-icon "dashboards" >}}
+
+2. Click the name of the dashboard that corresponds to your demo data bucket.
+
+## Demo data pricing and restrictions
+- InfluxDB Cloud demo data buckets are **free to use** and are **_not_ subject to
+  [Free Plan](/v2.0/account-management/pricing-plans/#free-plan) rate limits**.
+- Demo data buckets are **read-only**. You cannot write data into demo data buckets.
```
Comment (kelseiv) [inline content/v2.0/write-data/sample-data/demo-data.md:60]: drop line 57-60

File: content/v2.0/get-started.md [hunk 6]
```diff
@@ -458,6 +458,10 @@ For details, see [Scrape data](/v2.0/write-data/scrape-data/).
 For information about using the InfluxDB v2 API, `influx` CLI, and client libraries to write data,
 see [Write data to InfluxDB](/v2.0/write-data/).
 
+#### Demo data
+If using **{{< cloud-name "short" >}}**, [add a demo data bucket](/v2.0/write-data/sample-data/demo-data/)
```
Comment (kelseiv) [inline content/v2.0/get-started.md:462]: add **free** demo data to explore.

File: content/v2.0/write-data/sample-data/_index.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+  v2_0:
+    parent: Write data
+weight: 105
+---
+
+Use sample data to familiarize yourself with time series data and InfluxDB.
+InfluxData provides many sample time series datasets to use with **InfluxDB** and **InfluxDB Cloud**.
+
+{{< children >}}
+
```
Comment (kelseiv) [inline content/v2.0/write-data/sample-data/_index.md:17]: To explore and learn time series and InfluxDB, use:
- Sample data sets:
    - Air sensor....
    - Bird...
- (InfluxDB Cloud only) [Demo data buckets](...)

File: content/v2.0/write-data/sample-data/demo-data.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+products: [cloud]
+---
+
+Use **InfluxDB Cloud** demo data buckets for quick and easy access to different
+types of demo data that let you explore and familiarize yourself with InfluxDB Cloud.
+
+- [Add a demo data bucket](#add-a-demo-data-bucket)
+- [Explore demo data](#explore-demo-data)
+- [Visualize demo data](#visualize-demo-data)
+- [Demo data pricing and restrictions](#demo-data-pricing-and-restrictions)
```
Comment (kelseiv) [inline content/v2.0/write-data/sample-data/demo-data.md:20]: link back to sample data as another way "to learn and explore"

File: content/v2.0/write-data/sample-data/_index.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+### Air sensor sample data
+The air sensor sample data is designed to represent an "Internet of Things" (IoT) use case
```
Comment (kelseiv) [inline content/v2.0/write-data/sample-data/_index.md:21]: Air sensor data represents...

File: content/v2.0/write-data/sample-data/_index.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+<a class="btn" href="https://github.com/influxdata/influxdb2-sample-data/tree/master/air-sensor-data" target="\_blank">
+  <span class="icon-github"></span> View air sensor sample data
+</a>
+
+_Used in [Query SQL data sources](/v2.0/query-data/flux/sql/)._
+
+---
+
+### Bird migration sample data
+This sample data is adapted from the [Movebank: Animal Tracking](https://www.kaggle.com/pulkit8595/movebank-animal-tracking)
```
Comment (kelseiv) [inline content/v2.0/write-data/sample-data/_index.md:34]: Bird migration data is adapted from the [Movebank: Animal Tracking on kaggle](...) and represents bird migratory...

File: content/v2.0/write-data/sample-data/_index.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+dataset available on [kaggle.com](https://kaggle.com).
+The dataset as a whole represents animal migratory movements throughout 2019.
+The data schema meets the requirements of the
```
Comment (kelseiv) [inline content/v2.0/write-data/sample-data/_index.md:37]: Because the data schema meets, you can work with geo....

**Pull request merged**